### PR TITLE
Add start jupyter cm

### PR DESCRIPTION
**Pull request by ericpre**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/start_jupyter_cm/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  home: https://github.com/hyperspy/start_jupyter_cm
+  license: BSD
+  license_family: BSD
+  license_file: LICENSE
+  summary: Add entries to start Jupyter from context menu.
+
+extra:
+  recipe-maintainers:
+    - ericpre
+    - francisco-dlp
```
Comment (ericpre) [inline recipes/start_jupyter_cm/meta.yaml]: @francisco-dlp: can you confirm you are happy to be a maintainer of this recipe?

Comment (francisco-dlp) [reply recipes/start_jupyter_cm/meta.yaml]: Yes, I am.

**Comment by scopatz** [comment]
This seems to fail to build on windows

**Comment by ericpre** [comment]
> This seems to fail to build on windows

Yes, this should be due to some leftover code, which has been now removed in https://github.com/hyperspy/start_jupyter_cm/pull/10.

We will do a release very soon.

**Comment by scopatz** [comment]
Can you skip windows for now so that CI passes? You can always add it back to the feedstock

**Comment by ericpre** [comment]
Yes, good idea, done!

**Pull request merged**